docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -12,9 +12,15 @@
 The Core ML Tools [Unified Conversion API](unified-conversion-api) produces Core ML models for iOS 13, macOS 10.15, watchOS 6, tvOS 13 or newer deployment targets. If your primary deployment target is iOS 12 or earlier, you can find limited conversion support for PyTorch models via the [onnx-coreml](https://github.com/onnx/onnx) package.
 ```
 
-## Generate a TorchScript Version
+To export a model from PyTorch to Core ML, there are 2 steps:
+1. Capture the PyTorch model graph from the original torch.nn.Module, via [`torch.jit.trace`](https://pytorch.org/docs/stable/generated/torch.jit.trace.html) or [`torch.export.export`](https://pytorch.org/docs/stable/export.html#torch.export.export).
+2. Convert the PyTorch model graph to Core ML, via the Core ML Tools [Unified Conversion API](unified-conversion-api).
 
-[TorchScript](https://pytorch.org/docs/stable/jit.html) is an intermediate representation of a PyTorch model. To generate a TorchScript representation from PyTorch code, use PyTorch's JIT tracer ([`torch.jit.trace`](https://pytorch.org/docs/stable/generated/torch.jit.trace.html)) to _trace_ the model, as shown in the following example:
+Core ML Tools has been supporting the conversion from torch.jit.trace graph since Core ML Tools 4, so this the more mature and stable path. The conversion from torch.export graph is newly added in Core ML Tools 8.0, currently in beta state, in line with the export API status in PyTorch. As of Core ML Tools 8.0, representative models such as mobile bert, resnet, vit, [mobile net](convert-a-torchvision-model-from-pytorch), [deep lab](convert-a-pytorch-segmentation-model), [openelm](convert-openelm) can be converted, and total torch op translation test coverage is roughly ~60%. We would recommend to start trying the torch.export path, as [PyTorch no longer offers new feature development for torch.jit.trace](https://github.com/pytorch/pytorch/issues/103841#issuecomment-1605017153).
+
+## Obtain the Original PyTorch Model
```

Comment:
Before this, add a sentence, sth like:

"Now let's take a look at how to convert to Core ML through an example."

Reply:
Added as suggested
```
Now let us take a closer look at how to convert from PyTorch to Core ML through an example.
```